core/indexer/processTransactions.go

```diff
@@ -69,8 +69,9 @@ func (tdp *txDatabaseProcessor) prepareTransactionsForDatabase(
 		gasUsed := big.NewInt(0).SetUint64(tx.GasPrice)
 		gasUsed.Mul(gasUsed, big.NewInt(0).SetUint64(tx.GasLimit))
 		gasUsed.Sub(gasUsed, rec.Value)
+		gasUsed.Div(gasUsed, big.NewInt(0).SetUint64(tx.GasPrice))
 
-		tx.GasUsed = gasUsed.String()
+		tx.GasUsed = gasUsed.Uint64()
```

Comment:
gasUsed for normal transactions can be computed via ComputeGasLimit(tx) txFeeHandler